File: week6/mario.py
print("Hello world! \nWelcome to mario.")
height = int(input("How many levels do you want the mountain to be? "))
print(f"you entered: {height}")
for level in range(height):
    print(" "*(height-(level+1))  +   (level+1)*"#"   + " " + (level+1)*"#" )

File: week1/mario.c
#include <stdio.h>

void printTopdownBlocks(int levelsPrinted, int mountainHeight);

int main()
{
    int height;

    printf("Enter height for mario mountain.\n");
    scanf("%d", &height);
    printf("Height entered: %d\n", height);
    printTopdownBlocks(0, height);
    
    return 0;
}

 void printTopdownBlocks(int levelsPrinted, int mountainHeight){
    if(levelsPrinted < mountainHeight){
        for(int i = 0; i<=(mountainHeight - levelsPrinted); i++){   
            printf(" ");
        }
        for(int i = 0; i<=levelsPrinted; i++){   
            printf("#");
        }
        printf(" ");
        for(int i = 0; i<=levelsPrinted; i++){   
            printf("#");
        }
        printf("\n");
        printTopdownBlocks(levelsPrinted+1, mountainHeight);
    } 
    return;
 }
